src/config/oauth.json

```diff
@@ -4,6 +4,10 @@
   "api_server_base_path": "{{ .Env.K8S_API_SERVER_BASE_PATH }}",
   "ws_api_server": "{{ .Env.WS_K8S_API_SERVER }}",
   "openshift_console_url": "{{ .Env.OPENSHIFT_CONSOLE_URL }}",
+  "forge_api_url": "{{ .Env.FABRIC8_FORGE_API_URL }}",
```

Comment:
@jstrachan are you still good to have the runtime-console merged into fabric8-ui?